README.md

```diff
@@ -5,6 +5,7 @@ Example of use "Double Dispatch" pattern to avoid `static_cast` restrictions
 ## Platform ##
 
 * Linux
+* macOS
```

Comment:
@Morheit , did you test in this platfom?

Reply:
Yes. This branch was developed and tested on macOS